### ISSUE TITLE
context in composer

### ISSUE THREAD

**Issue by morta5** [mastodon/mastodon-android#11]
It would be great to have more context in the composer when answering to a toot as you often have to think about what you're answering to when writing one. 
It could look a bit like it does on Twitter(or even better, by showing the whole thread):
![Screenshot_20220414-143228](https://user-images.githubusercontent.com/66527936/163391529-10fa5e4f-50c7-4338-a0a6-2e9278e45c43.png)

**Comment by realpixelcode** [comment]
Yeah, this would be great! For reference, this is what the feature looks like in Tusky:

![Screenshot-2022-04-20-01-06-17.png](https://user-images.githubusercontent.com/52963327/164235603-a600b48c-2ffe-4090-8cb4-5e00d9bfe63b.png)
